compass/ocean/tests/drying_slope/viz/__init__.py

```diff
@@ -108,11 +110,10 @@ def _plot_ssh_time_series(self, outFolder='.'):
         fig, _ = plt.subplots(nrows=naxes, ncols=1, figsize=figsize, dpi=100)
 
         for i in range(naxes):
-            ax = plt.subplot(naxes, 1, i+1)
+            ax = plt.subplot(naxes, 1, i + 1)
             ds = xarray.open_dataset(ncFilename[i])
```

Comment:
A very optional comment: sInce you're switching numpy->np you could do the same for xarray->xr